server/models/comments.js

```diff
@@ -32,7 +34,23 @@ const getCommentModel = (sequelize, DataTypes) => {
     });
 
     Comment.belongsTo(db.Articles, {
-      foreignKey: "articleId"
+      foreignKey: "articleId",
+      onDelete: "CASCADE"
+    });
+    Comment.hasMany(db.Reactions, {
+      scopes: {
```

Comment:
This looks like something out of NASA, I believe keeping simple things simple can make it much easy to follow. Is there any way we can make this more readable

Reply:
please check here for more information. https://sequelize.readthedocs.io/en/latest/docs/scopes/